### DISCUSSION TITLE
Student Developer pack approved but benefits not availed.

### DISCUSSION THREAD

**Discussion by shubz007** [community/community#135996] [GitHub Education] [answered]
### Hi everyone,

My Github developer pack was approved on 11th August and today it is 16th August and i Still haven't received the benefits .
![Screenshot 2024-08-16 212040](https://github.com/user-attachments/assets/802b1e12-c89f-491a-aafd-1a95c4f1ac0e)

**Comment by davevad93** [comment, marked as answer]
Hi @shubz007 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @shubz007 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135145, #135407

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
